cfme/tests/pod/test_appliance_crud.py

```diff
@@ -208,7 +208,7 @@ def test_crud_pod_appliance(temp_pod_appliance, provider, setup_provider):
         test_flag: podtesting
 
     Polarion:
-        assignee: None
+        assignee: izapolsk
```

Comment:
missing `caseimportance` on some of these. Also need to fill out initialEstimate